node-js-basics-101/node_modules/blob/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Create a blob builder even when vendor prefixes exist
+ */
+
+var BlobBuilder = global.BlobBuilder
+  || global.WebKitBlobBuilder
+  || global.MSBlobBuilder
```

Comment:
__[jshint]__
* Misleading line break before '||'; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->